statsmodels/regression/_tools.py

```diff
@@ -0,0 +1,105 @@
+from collections import namedtuple
+import numpy as np
+
+_MinimalWLSResults = namedtuple('_MinimalWLSResults',
```

Comment:
It is better to make this into simple Dummy/Holder/Bunch classes.
tuples are not forward compatible if we want to change what is returned.